.ruby-version

```diff
@@ -1 +1 @@
-2.1.2@freckler
+2.1.2@pecas
```

Comment:
@mauro-oto Could you remove `.ruby-version` from the repo and add it to `.gitignore`?